### DISCUSSION TITLE
790

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#961] [Comments]
# 790

다른 사람들이 쉽게 하거나 관심을 가지는 일상적인 것들에 대해 크게 관심이 없는 사람이라서 나 스스로가 어딘가 나사 하나가 빠져있다는 생각을 종종 한다.그래서 커리어 외에 다른 것에 대해 다른 사람들의 조언을 들을때면 크게 와닿지 못하고, 갸우뚱 할 때가 자주 있었다.결혼을 준비하면서도 앞으로의 내 결혼 생활을 어떻게 해야하나 여러 책이나 조언들을 들어도 추상적이여서 나 같이 뭔가가 부족한 사람에게는 머릿 속으로 잘 그려지지 않았다.어떤때면 어떻게 해라 라는 조언이 들릴때면 그럼 이럴땐 어떡하지? 저럴땐 어떡하지? 등의 생각이 자연스럽게 들어서 머릿속만 더 복잡해졌다.예전에 빈지노 & 미초바 부부의 연애/결혼 생활 영상을 보면서 "이 커플은 어떻게 이렇게 오래 만나고도 이쁘게 사랑할 수 있지?" 라며 되..

https://jojoldu.tistory.com/790

**Comment by jws1837** [comment]
저도 딱 와닿는 표현이었어요. 결혼생활을 커리어처럼.  좋은 가사를 쓰는 래퍼들의 표현력은 정말 대단한 거 같아요.!

**Comment by mag123c** [comment]
결혼 생활이라는 게 일처럼 잠시 떼어두고 쉬고 할 수 있는 부분이 아니니 평생 짊어지는 일이 되면 지칠 때 더 크게 지치고 힘들 것 같기도 합니다. 그래도 훌훌 털어내려고 노력하고 있습니다. 😢  평생 나를 위해, 서로를 위해 노력하며 사는 부부가 좋은 부부가 되리라 믿고 노력하며 살아가고 있습니다ㅋㅋ..  잘 읽었습니다 감사합니다
